tomls/omnibus-base-mainnet-andromeda/rewards/scbeth_perps_liquidations_rewards.toml

```
@@ -0,0 +1,46 @@
+[provision.RewardsDistributor_scbeth_liquidation_rewards]
+source = "synthetix-rewards-dist-ext:3.6.0"
+target = "synthetix-rewards-dist-ext:3.6.0@perps-scbeth"
+options.salt = "<%= settings.salt %>"
```

for all these we need to update the salt too

```suggestion
options.salt = "<%= settings.salt %>-scbeth"
```